karmagrambot/commands.py

```diff
@@ -36,7 +36,24 @@ def karma(bot: Bot, update: Update):
 
     update.message.reply_text(karma)
 
+def devil(bot: Bot, update: Update):
+    """Reply the user with some dumb text and the person with the lowest karma, the "devil".
+
+    Args:
+        bot: The object that represents the Telegram Bot.
+        update: The object that represents an incoming update for the bot to handle.
+
+    Returns:
+        Doesn't return, but answer the user with the one with the lowest karma.
```

Comment:
If a function does not return, describe its behaviour in a paragraph after the brief summary and omit the `Returns` section.